server/ngwmn/filters.py

```diff
@@ -0,0 +1,22 @@
+"""
+Jinja2 filters. Must be imported (via ngwmn.__init__) for them to register
+via the `app.template_filter` decorator.
+"""
+
+from urllib.parse import urljoin
+
+from . import app
+
+
+@app.template_filter('asset_url')
+def asset_url_filter(asset_src):
```

Comment:
Consider bringing over the tests for this from waterdataui. It's possible the two filters would diverge. We should probably also consider creating some py modules for common code between projects.